Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -170,7 +170,7 @@ public static AppDetails QueryAppDetails(string baseAppName, ConnectInfo connect
                 AppList appList = JsonUtility.FromJson<AppList>(appListJSON);
                 for (int i = 0; i < appList.InstalledPackages.Length; ++i)
                 {
-                    string appName = appList.InstalledPackages[i].PackageFullName;
+                    string appName = appList.InstalledPackages[i].Name;
```

Comment:
Fair point - we should have the full package name. Let me see if I can filter that all the way through.